Makefile

```diff
@@ -1,11 +1,24 @@
 # Copyright Contributors to the Open Cluster Management project
 
--include /opt/build-harness/Makefile.prow
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+VERSION ?= 0.0.1
 
-.PHONY: bundle-latest-image
-bundle-latest-image:
-	bash ./scripts/bundle-image-pickup.sh -s
+# For example, running 'make bundle-build bundle-push' will build
+# open-cluster-management.io/search-v2-operator-bundle:$VERSION.
+IMAGE_TAG_BASE ?= open-cluster-management.io/search-v2-operator
```

Comment:
I think we can remove the `-v2` to match the quay repo name.

Reply:
Removed `-v2`.